src/Consumer.php

```diff
@@ -42,6 +45,11 @@ public function setConsumerTag(string $value): void
         $this->consumerTag = $value;
     }
 
+    public function setMaxPriority(int $value): void
+    {
+        $this->maxPriority = $value ?? 1;
```

Comment:
value is not nullable, so `?? 1` is redundant

Reply:
Fixed